### PR TITLE
CASMMON-259 Create documentation for Observability Framework

### PR DESCRIPTION
**Pull request by sumithrb**
# Description

Documentation for Install and upgrade observability framework

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -77,7 +77,10 @@ Jenkinsfile.github                                       @Cray-HPE/csm-devops
 # IUF Items
 /operations/iuf/                                         @Cray-HPE/IUF
 
+#Observability Items
+/operations/observability                                @Cray-HPE/cray-monitoring
+
 # Include SSI on these CSM items.
-/install/                                                @Cray-HPE/docs-csm-reviewers @Cray-HPE/metal @Cray-HPE/ssi-reviewers
```
Comment (rustydb) [inline .github/CODEOWNERS:81]: The commit message does not indicate why this is changing at all for `/install`/, `/operations/`, and `/upgrade/`. It seems like a bad rebase/merge was done.

**Review by rustydb — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (sumithrb) [reply .github/CODEOWNERS:81]: Yes, it seems like the wrong merge. I have reverted the change and the only change required is added.

**Review by rustydb — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Review by rambabubolla — APPROVED** [review]

**Comment by deantroe** [comment]
@rustydb this PR should not have merged.  it undid a lot of the work I had done in https://github.com/Cray-HPE/docs-csm/commit/69ba606471df40c800b9ef18425419c0134758da

Can we please back this out ASAP?

**Pull request merged**